packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanTreeOffset.css

```diff
@@ -39,15 +39,11 @@ limitations under the License.
 }
 
 .SpanTreeOffset--indentGuide.is-active {
-  /* The size of the indentGuide is based off of the iconWrapper */
-  padding-right: calc(0.5rem + 11px);
-  border-left: 0px;
+  border-left: 3px solid darkgrey;
```

Comment:
I think you can get away with just `border-color: darkgrey` to avoid reproducing the other two characteristics (`3px` and `solid`)